src/js/components/Carousel/StyledCarousel.js

```diff
@@ -0,0 +1,77 @@
+import styled, { css, keyframes } from 'styled-components';
+import { Box } from '../Box';
+import { widthStyle, heightStyle } from '../../utils';
+
+const StyledCarouselContainer = styled(Box)`
+  position: relative;
+  display: block;
+  overflow: hidden;
+  height: 100%;
+  ${(props) => props.height && heightStyle(props.height, props.theme)}
+  ${(props) => props.width && widthStyle(props.width, props.theme)}
+  ${(props) => props.containerHeight && `height: ${props.containerHeight}px`};
+  ${(props) => props.containerWidth && `width: ${props.containerWidth}px`};
+`;
+
+const StyledCarouselChild = styled.div`
+  position: relative;
+  max-width: 100%;
+  ${(props) =>
+    !props.noDimensions &&
+    css`
+      width: 100%;
+      height: 100%;
+    `}
+  float: left;
+  margin-right: -100%;
+  display: ${(props) => props.displayProp};
+  animation: ${(props) =>
+    props.animation
+      ? css`
+          ${props.animation}
+          ${props.animationDuration / 1000}s
+        ease-in-out
+        `
+      : `none`};
+  animation-fill-mode: both;
```

Comment:
This was added so the translate won't pop back to where it was once the animation completes